testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -463,167 +421,172 @@ def test_coordinates(self):
                 ts_orig.frame))
 
 
-class TestDCDCorrel(TestCase):
```

Comment:
Technically, this should just be marked as `@knowfailure` or `@skip` and then removed together with #1373. Or merge #1373 *first* and rebase this PR.